splot/_viz_value_by_alpha_mpl.py

```diff
@@ -215,13 +213,29 @@ def vba_choropleth(x, y, gdf, cmap='GnBu',
     
     >>> fig = plt.figure(figsize=(15,10))
     >>> ax = fig.add_subplot(111)
-    >>> vba_choropleth(x, y, gdf, divergent=True,
+    >>> vba_choropleth('HOVAL', 'CRIME', gdf, divergent=True,
     ...                alpha_mapclassify=dict(classifier='quantiles', k=5),
     ...                rgb_mapclassify=dict(classifier='quantiles', k=5),
     ...                legend=True, ax=ax)
     >>> plt.show()
 
     """
+    
+    if(type(x_var) is not str):
```

Comment:
I would try:

`if isinstance(x_var, str):
     x = np.array(gdf[x_var])
else
 x = x_var`

and the same for y_var, reduces the ifs a little :)

Reply:
of course also including your test :)

Reply:
Nice... is there a to include an additional step in this if? I mean, if the user passes an x_var which is a string but it is misspelled.